tools/example.py

```diff
@@ -199,7 +199,11 @@ async def main() -> None:
         gen = 3
 
     if args.debug:
-        logging.basicConfig(level="DEBUG", force=True)
+        logging.basicConfig(level=logging.DEBUG)
+        if args.gen1:
+            logging.getLogger("aioshelly.rpc_device").setLevel(logging.INFO)
+        else:
```

Comment:
The current implementation doesn't match the PR description, if you don't specify any gen you disable debug logging for Gen1, should check that no Gen is in params:
```suggestion
        elif args.gen2 or args.gen3:
```